src/javascript/common/champion.js

```diff
@@ -53,6 +53,7 @@ const Champion = (function() {
     };
 
     const beforeContentChange = () => {
+        Slider.destroy();
```

Comment:
This would be called in every page. In order to run it only on home page, please add `home` to `pages_map` and call it there.